reviews/utils/review_actions.py

```diff
@@ -210,3 +212,63 @@ def action_url(self):
     def description(self):
 
         return _('Verander aangestelde commissieleden')
+    
+class SendConfirmation(ReviewAction):
+
+    def is_available(self):
+        '''Only the secretary is able to send the confirmation letter and/or change the date.
+        The review needs to be closed and have an approved status.'''
+
+        user = self.user
+        review = self.review
+
+        user_groups = user.groups.values_list("name", flat=True)
+        if not settings.GROUP_SECRETARY in user_groups:
+            return False
+        
+        if review.stage < review.CLOSED:
+            return False
+        
+        if not review.continuation in [review.GO, review.GO_POST_HOC]:
+            return False
+        
+        return True
+    
+    def action_url(self):
+
+        return reverse('proposals:confirmation', args=(self.review.proposal.pk,))
+    
+    def description(self):
+        send_letter = _('Bevestigingsbrief versturen')
+        change_date = _('Datum van bevestigingsbrief aanpassen')
+        return send_letter if self.review.proposal.date_confirmed is None else change_date
+
+class HideReview(ReviewAction):
+    '''This class should lead to the archive_hide url, but the hide functionality does currently not work properly/ is not in use.
```

Comment:
Try to keep your line lengths reasonable. We haven't yet set a hard rule in our team, but the Python style recommendations suggest a maximum of 79 characters.